docs/faq.md

```diff
@@ -0,0 +1,71 @@
+
+9. Does Promscale support backfill?
+
+    Yes. Promscale support backfill of old data by sending older date to Promscale using the Prometheus [remote write API](https://prometheus.io/docs/prometheus/latest/storage/#remote-storage-integrations). This requires creating a custom application that can send requests to that API. We have plans in the near future to expose an SQL API to ingest data into Promscale.
+
+10. How do I store existing data from Prometheus to Promscale?
+
+    Right now, there is no good way to migrate existing data from your Prometheus instance to Promscale. We will be working on a tool which would enable you to do so in the near future.
+
+11. Why doesn’t Prometheus show Promscale data when I use the PromQL dashboard?
+
+    If you have backfilled data which is not ingested via Prometheus itself, you will need to enable the `read_recent` option for Promscale remote read endpoint ([details here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read)).
+
+12. How do I make the maximum use of Promscale in terms of efficient storage use? 
+
+    Since all your Prometheus data will be stored in Promscale as well, we suggest keeping the Prometheus retention period to 6 hours so your Prometheus instance will not take up unnecessary storage space.
+
+13. Are recording rules supported by Promscale? 
+
+    Yes. Prometheus manages the rules and Promscale will create the series on command from Prometheus.
+
+14. How can I find out more about Promscale?
+
+    If you have any questions, please join the #prometheus channel on [TimescaleDB Slack](https://slack.timescale.com/), or the [Promscale Users Google Group](https://groups.google.com/forum/#!forum/promscale-users). You can also join our users & community meeting ([details here](https://tsdb.co/promscale-agenda)).
+
+15. Does Promscale need a database superuser to run?
+
+    No, Promscale does not need to run as superuser when ingesting or querying data. To setup or update the schema, you could run the Promscale with the `-migrate=only` flag using a superuser which will install the update the schema and install the necessary extensions and quit. You can also avoid needing superuser access by installing the TimescaleDB extension manually before starting the connector.
+
+16. How do I downsample data with Promscale?
+
+    Right now, the easiest way is to use [Prometheus recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) to downsample the wanted data and set a longer retention period on the metric data which was created by the recording rules. Note that recording rules only allow you to downsample data after the rule has been created and, as of right now, it cannot be applied to pre-existing data.
```

Comment:
> longer retention period on the metric data

I think you meant evaluation interval per **rule group**?

Also, if the aim is retention interval here, then the storage should be specified.

Reply:
Yeah, let me change that to `set a longer evaluation interval for the rule group`. Does that sound OK?

Reply:
Yes, that is good.